plugins/com.google.cloud.tools.eclipse.preferences/plugin.xml

```diff
@@ -16,5 +17,14 @@
             class="com.google.cloud.tools.eclipse.preferences.PreferenceInitializer">
       </initializer>
    </extension>
+   <extension
+         point="com.google.cloud.tools.eclipse.preferences.areas">
+      <area
+            page="com.google.cloud.tools.eclipse.preferences.main"
+            title="Usage Statistics"
+            class="com.google.cloud.tools.eclipse.preferences.AnalyticsOptInArea"
+            preferences="instance://com.google.cloud.tools.eclipse.preferences">
```

Comment:
Analytics preference store was previously configuration-scoped. The instance scope is per workspace, so I believe the configuration scope is the right choice?